HPXMLtoOpenStudio/measure.rb

```diff
@@ -1929,21 +1942,46 @@ def self.add_lighting(runner, model, epw_file, spaces)
   end
 
   def self.add_pools_and_hot_tubs(runner, model, spaces)
+    occ_calc_type = @hpxml.header.occupancy_calculation_type
+    unit_type = @hpxml.building_construction.residential_facility_type
+    cfa = @hpxml.building_construction.conditioned_floor_area
+    nbeds = @hpxml.building_construction.number_of_bedrooms
+    noccs = @hpxml.building_occupancy.number_of_residents
+    if occ_calc_type == 'operational'
+      adj_factor = HPXMLDefaults.get_misc_luls_adjustment_factor(unit_type, nbeds, noccs, cfa)
```

Comment:
Rather than calculate the same factor multiple places across the source code, just calculate it once and reuse it. For example, you could calculate it in hpxml_defaults.rb and stored it on `hpxml.header.additional_properties`.